target/linux/mediatek/dts/mt7986a-tplink-tl-xdr4288.dts

```diff
@@ -15,22 +15,22 @@
 
 		port@0 {
 			reg = <0>;
-			label = "lan1";
+			label = "lan4";
 		};
 
 		port@1 {
 			reg = <1>;
-			label = "lan2";
+			label = "lan3";
 		};
 
 		port@2 {
 			reg = <2>;
-			label = "lan3";
+			label = "lan2";
```

Comment:
We can not really change this any more. 

This is part of openwrt 23.05 and users might have configurations referencing `lan2` for example. If they upgrade it would be a different port.

Reply:
Ok, I removed this change.